test/utils.sh

```diff
@@ -154,3 +156,18 @@ parse_test_output() {
     exit 1
   fi
 }
+
+handle_error()
```

Comment:
Where is this function called?

Reply:
https://github.com/redhat-appstudio/build-definitions/pull/570/files#diff-0240a91e2ace74bf029b4153dcb9ba968842bb5a69706780a9d717d7ca8533b3R56

Reply:
A comment is added for the usage of this function. Thanks.